src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -273,6 +273,7 @@
       [:div {:style {:float "left"}}
        (make-field entity "createDate" "Created: " false common/format-date)
        (make-field entity "entityType" "Entity Type: " false)
+       (make-field entity "managers" "Owners: " false (partial clojure.string/join ", "))
```

Comment:
my bad - must have been an error in my environment. I see "managers" in the json response now, all good!